theme/book.js

```diff
@@ -461,6 +463,10 @@ function playground_text(playground) {
         el.addEventListener('click', toggleSection);
     });
 
+    Array.from(sidebarDraftToggles).forEach(function (el) {
+        el.addEventListener('click', toggleSection);
+    });
+
```

Comment:
add toggle event listener for sidebar div types which are not toggled by default